superset/examples/birth_names.py

```diff
@@ -206,26 +193,16 @@ def create_slices(tbl: SqlaTable, admin_owner: bool) -> Tuple[List[Slice], List[
         ],
     }
 
-    admin = get_admin_user()
-    if admin_owner:
-        slice_props = dict(
-            datasource_id=tbl.id,
-            datasource_type=DatasourceType.TABLE,
-            owners=[admin],
-            created_by=admin,
-        )
-    else:
-        slice_props = dict(
-            datasource_id=tbl.id,
-            datasource_type=DatasourceType.TABLE,
-            owners=[],
-            created_by=admin,
-        )
+    slice_kwargs = {
+        "datasource_id": tbl.id,
+        "datasource_type": DatasourceType.TABLE,
+        "owners": [],
+    }
```

Comment:
Replace "props" with "kwargs" and `dict()` with `{}` to make this more pythonic.